src/main/java/com/mineinabyss/mobzy/pathfinders/hostile/ItemThrowable.kt

```diff
@@ -0,0 +1,10 @@
+package com.mineinabyss.mobzy.pathfinders.hostile
+
+import org.bukkit.entity.LivingEntity
+import org.bukkit.inventory.ItemStack
+
+interface ItemThrowable {
```

Comment:
i would like to think their are more atributes about how items should be thrown?
also only targeting living entities seems questionable.